framework/test/gtest/AnyMapTest.cpp

```diff
@@ -136,15 +135,16 @@ TEST(AnyMapTest, IteratorTest)
 
 TEST(AnyMapTest, AnyMap)
 {
-    AnyMap::ordered_any_map o;
```

Comment:
I would like to see explicit tests for initializer_list support instead of only replacing the existing tests with the intializer_list supported usage.

Reply:
Done.